pyEpiabm/pyEpiabm/core/place.py

```diff
@@ -105,14 +126,38 @@ def remove_person(self, person: Person):
         if person not in self.persons:
             raise KeyError("Person not found in this place")
         else:
+            group_index = self.get_group_index(person)
+            self.person_groups[group_index].remove(person)
             person.remove_place(self)
             self.persons.remove(person)
 
-    def empty_place(self):
-        """Remove all people from place. For example
+    def get_group_index(self, person):
+        """Get the group of a person in the place.
+
+        :param person: Person associated with group
+        :type person: Person
+        """
+        if person not in self.persons:
+            raise KeyError("Person not found in this place")
+        else:
+            place_list = [i[0] for i in person.places]
+            ind = place_list.index(self)
+        return person.places[ind][1]
+
+    def empty_place(self, groups_to_empty: list = [0]):
```

Comment:
I'm not convinced that this is a good default behavior. I would either require that a group be passed, or make the default be to empty all groups.

Reply:
Defaulted to empty groups